.travis.yml

```diff
@@ -13,13 +13,15 @@ env:
   - TEST_SUITE=doxygen_documentation
 
 before_install:
-  - docker pull dalg24/clang-format
+  - docker pull dalg24/clang-format:breathe
 
 script:
+  - ci_env=`bash <(curl -s https://codecov.io/env)`
```

Comment:
I could have gone by with `-e TRAVIS` but I figured I would just forward all the env.  We were not using any of the [Travis CI env variables](https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables) previously.  Do you want me to only pass the one we use?